release/docker/build-image-multi-arch.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+set -e
+
+# Import libs
+. ../../lib/shell/file_management.sh
+
+# Variable
+OLDIFS=$IFS
+BUILDER_NUM=`date +%s`
+BUILDER_NAME="multiarch_${BUILDER_NUM}"
+DIR=""
+
+function usage() {
+    echo ""
+    echo "This script is used to build the OpenSearch Docker image with single architecture (x64 or arm64). It prepares the files required by the Dockerfile in a temporary directory, then builds and tags the Docker image."
+    echo "--------------------------------------------------------------------------"
+    echo "Usage: $0 [args]"
+    echo ""
+    echo "Required arguments:"
+    echo -e "-v VERSION\tSpecify the OpenSearch version number that you are building, e.g. '1.0.0' or '1.0.0-beta1'. This will be used to label the Docker image. If you do not use the '-o' option then this tool will download a public OPENSEARCH release matching this version."
+    echo -e "-f DOCKERFILE\tSpecify the dockerfile full path, e.g. dockerfile/opensearch.al2.dockerfile."
+    echo -e "-p PRODUCT\tSpecify the product, e.g. opensearch or opensearch-dashboards, make sure this is the name of your config folder and the name of your .tgz defined in dockerfile."
+    echo -e "-a ARCHITECTURE\tSpecify the multiple architecture you want to add to the multi-arch image, separate by comma, e.g. 'x64,arm64'."
+    echo -e "-r REPOSITORY\tSpecify the docker repository name in the format of '<Docker Hub RepoName>/<Docker Image Name>:<Tag Name>', due to multi-arch image either save in cache or directly upload to Docker Hub Repo, no local copies."
+    echo ""
+    echo "Optional arguments:"
+    echo -e "-o TARBALL\tSpecify multiple opensearch or opensearch-dashboards tarballs, use the same order as the input for '-a' param, e.g. 'opensearch-1.0.0-linux-x64.tar.gz,opensearch-1.0.0-linux-arm64.tar.gz'. You still need to specify the version - this tool does not attempt to parse the filename."
+    echo -e "-h\t\tPrint this message."
+    echo "--------------------------------------------------------------------------"
+}
+
+function cleanup_docker_buildx() {
+    # Cleanup docker buildx
+    echo -e "\n* Cleanup docker buildx"
+    docker buildx use default
+    docker buildx rm $BUILDER_NAME > /dev/null 2>&1
+}
+
+function cleanup_all() {
+    cleanup_docker_buildx
+    File_Delete $DIR
+}
+while getopts ":ho:v:f:p:a:r:" arg; do
+    case $arg in
+        h)
+            usage
+            exit 1
+            ;;
+        o)
+            TARBALL=`realpath $OPTARG`
+            ;;
```

Comment:
Minor can ignore: `o` is quite confusing. Usually specifies the output file. Can we use something else?

Reply:
What would you suggest? I dont think -t would be better.

Reply:
`-t` works. @camerski any thoughts?

Reply:
Yeah, `-t` seems more natural.

Reply:
Point taken, will make the changes and commit again.

Reply:
Changed.